spec/support/comment_shared_examples.rb

```diff
@@ -100,4 +100,35 @@
       end
     end
   end
+
+  describe 'local caching', js: true do
+    let!(:original_page_with_comment_path) { page.current_path }
+
+    context 'when comment is not saved' do
+      it 'prefills textarea with cached value' do
+        within 'form[data-behavior~=add-comment]' do
+          fill_in 'comment[content]', with: 'test comment'
+          sleep 1 # Needed for setTimeout function in local_auto_save.js
```

Comment:
Is there anything we can do here besides a sleep? If it was in the editor I've been using
`find('.textile-preview', text: 'New info', wait: 1)` which waits for the `textchange` event to produce a change on the page. Is there anything we could use to identify the event complete instead if just `sleep`ing?

Reply:
I tried `fill_in 'comment[content]', with: 'test comment', wait: 1` and it's not working. Can't seem to get any results on Google as well.